### ISSUE TITLE
Prebuilt binaries for i686, armhf and aarch64

### ISSUE THREAD

**Issue by StevenSeifried** [pine64/blisp#32]
I create the [blisp-bin package in the AUR](https://aur.archlinux.org/packages/blisp-bin). Currently, it's only for `x86_64`, but I also want made it compatible to `i686`, `armhf` and `aarch`.

Therefor, it would be nice when there were also prebuilt binaries for `i686`, `armhf` and `aarch` available in the releases section

**Comment by River-Mochi** [comment]
steven  are you going to add something on the readme page to let people know  how to get the blisp package in AUR?

**Comment by StevenSeifried** [comment]
> steven are you going to add something on the readme page to let people know how to get the blisp package in AUR?

Yes, I will do that. I wanted to wait until `blisp-bin` is available for `i686`, `armv7h` and `aarch64` as well.

**Comment by River-Mochi** [comment]
> Yes, I will do that. I wanted to wait until `blisp-bin` is available for `i686`, `armv7h` and `aarch64` as well.

 that might take a while as this is all volunteer time :)

**Comment by River-Mochi** [comment]
> This project is, however, developer - limited and should try to recruit more worker bees.

anyone is allowed to improve blisp project and just submit PR for what they like. Marek is very busy and he working on other project currently I think.

**Comment by River-Mochi** [comment]
> I create the [blisp-bin package in the AUR](https://aur.archlinux.org/packages/blisp-bin).

perhaps do a PR  now for directions on Readme to get the one  AUR  package you made. since it might be weeks or never that those other binaries are made since  Gamiee is busy and no one else who knows how to do it has time or stepped up to do it.
